src/Campaigns/ServiceProvider.php

```diff
@@ -100,6 +101,7 @@ private function registerActions(): void
     {
         Hooks::addAction('givewp_campaign_deleted', DeleteCampaignPage::class);
         Hooks::addAction('givewp_donation_form_creating', FormInheritsCampaignGoal::class);
+        Hooks::addAction('givewp_campaign_page_created', AssociateCampaignPageWithCampaign::class);
```

Comment:
Out of curiosity, what order are we creating these objects (`Campaign` & `CampaignPage`) in?

Reply:
Ah okay, thanks